keyboards/z150_blackheart/keymaps/default/keymap.c

```diff
@@ -0,0 +1,73 @@
+#include QMK_KEYBOARD_H
+
+// Fillers to make layering more clear
+#define ______ KC_TRNS
+#define XXXXXXX KC_NO
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	LAYOUT(
```

Comment:
Looks like you have too many keys here.

Reply:
Yeah, was a mistake I hadn't noticed before, fixed now.